DeezyMatch/candidateRanker.py

```diff
@@ -363,12 +394,13 @@ def candidate_ranker(input_file_path="default", query_scenario=None, candidate_s
             collect_neigh_pd = collect_neigh_pd.sort_values(by="dl_match", ascending=False)[:num_candidates]
     
         for i_row, row in collect_neigh_pd.iterrows():
-            if not pretrained_model_path in [False, None]:
-                mydict_dl_match[row["s2_orig"]] = round(row["dl_match"], 4)
-                mydict_dl_1_minus_match[row["s2_orig"]] = 1. - round(row["dl_match"], 4)
-            else:
-                mydict_dl_match[row["s2_orig"]] = row["dl_match"]
-                mydict_dl_1_minus_match[row["s2_orig"]] = 1. - row["dl_match"]
+            if use_predict == True:
+                if not pretrained_model_path in [False, None]:
+                    mydict_dl_match[row["s2_orig"]] = round(row["dl_match"], 4)
+                    mydict_dl_1_minus_match[row["s2_orig"]] = 1. - round(row["dl_match"], 4)
+                else:
```

Comment:
Why rounding depends on `if not pretrained_model_path in [False, None]`

Reply:
Good question, I am not sure... I think that was already there, but I cannot see why we would need this distinction.